commands/git-finish

```diff
@@ -29,19 +29,39 @@ if [[ $BRANCH = 'master' || $BRANCH = 'develop' ]]; then
   exit 255
 fi
 
+if [[ $DEVELOP_EXISTS -eq 0 ]]; then
+  git checkout develop
+  git pull origin develop
+fi
+
 if [[ $TYPE = 'hotfix' || ${BRANCH:0:7} = "hotfix/" || $DEVELOP_EXISTS -ne 0 ]]; then
   git checkout master
   git pull origin master
-  git merge --no-ff $BRANCH
-  [[ $1 = 'push' || $2 = 'push' ]] && git push origin master
+fi
+
+git checkout $BRANCH
+
+if [[ $DEVELOP_EXISTS -eq 0 ]]; then
+  git merge --no-ff develop
```

Comment:
If we do this maybe these (2) pre-merges should be done without `--no-ff`? Need to see the different histories to know for sure.